src/components/AnnualReport/ARDate.vue

```diff
@@ -2,28 +2,29 @@
   <v-card flat class="ar-date-container">
     <div class="timeline">
       <label>Annual Report Date</label>
-      <span class="date ar-date">{{this.toReadableDate(this.nextARDate)}}</span>
+      <span class="date ar-date">{{toReadableDate(nextARDate)}}</span>
       <label>Filing Date</label>
-      <span class="date file-date">Today ({{this.toReadableDate(new Date())}})</span>
+      <span class="date file-date">Today ({{toReadableDate(currentDate)}})</span>
     </div>
   </v-card>
 </template>
 
 <script lang="ts">
 // Libraries
-import { Component } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
 import { mapState } from 'vuex'
 
 // Mixins
 import { DateMixin } from '@/mixins'
 
 @Component({
   computed: {
-    ...mapState(['nextARDate'])
+    ...mapState(['nextARDate', 'currentDate'])
   }
 })
-export default class ArDate extends DateMixin {
+export default class ArDate extends Mixins(DateMixin) {
```

Comment:
This is functionally equivalent to the previous code, but now it's consistent across modules.